java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/containerregistry/ContainerRegistryDeployer.java

```diff
@@ -98,6 +98,8 @@ public void deployArchive(Path deployDir, Path sourcePath, Path logsPath, Set<St
             createImages(imageData, sourcePath, logsPath, imageNameHashCallback);
 
         } finally {
+            // TODO: Why is this deleting the file tree here?
+            Log.warnf("### Deleting %s", imageData.getArtifactsPath());
             FileUtil.deleteRecursive(imageData.getArtifactsPath());
```

Comment:
This looks like it came from https://github.com/redhat-appstudio/jvm-build-service/pull/260